pkg/clusterresource/openstack.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/openshift/hive/pkg/constants"
 )
 
+const (
+	computeFlavor   = "m1.large"
+	masterFlavor    = "ci.m4.xlarge"
+	externalNetwork = "provider_net_shared_3"
+)
```

Comment:
In case of Openstack, if install config template is not provided, then the generated config has these settings and network fields missing. I wanted a minimal CDC on user part, and I saw that these default settings were applied in hiveutil. But now that it is clear that hiveutil is for development only and these settings are probably unique to our Openstack, I will remove this change.